mesos_master/manifest.json

```diff
@@ -5,8 +5,7 @@
   "categories": [
     "configuration & deployment",
     "containers",
-    "orchestration",
-    "log collection"
+    "orchestration"
```

Comment:
why are removing the log collection tag on this one?

Collecting logs from mesos container is supported right?